.gitlab-ci.yml

```diff
@@ -67,3 +67,9 @@ clang-6.0:
         CONAN_CLANG_VERSIONS: "6.0"
     <<: *build-template
 
+clang-7:
+    image: lasote/conanclang7
```

Comment:
I thought we were only going to use conanio containers. Why didn't use use conanio/clang7 as on travis?

Reply:
Indeed, this is wrong. Seems I didn't review my rebase :man_facepalming: 
Fix follows …